script/deployParameters/arbitrum-goerli.json

```diff
@@ -2,6 +2,8 @@
   "permit2": "0x000000000022D473030F116dDEE9F6B43aC78BA3",
   "weth9": "0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3",
   "seaport": "0x0000000000000000000000000000000000000000",
+  "seaportV2": "0x0000000000000000000000000000000000000000",
```

Comment:
would probably rather it said `V1_2` or something - as this is not V2 yet

Reply:
Do you want me to apply this to the current branch for SeaportV4?